Index: pegasus/Schemas/Pegasus/InterOp/VER20/PG_ProviderModule20.mof
===================================================================
--- pegasus.orig/Schemas/Pegasus/InterOp/VER20/PG_ProviderModule20.mof	2009-03-18 09:49:14.000000000 +0100
+++ pegasus/Schemas/Pegasus/InterOp/VER20/PG_ProviderModule20.mof	2009-03-18 09:50:00.000000000 +0100
@@ -67,14 +67,28 @@
         "which this provider module must be invoked (regardless of which "
         "user requests an operation.)  This value MUST be set to a non NULL "
         "value when UserContext is set to a value of 3 (\"Designated "
         "User\").  For all other values of UserContext, the "
         "DesignatedUserContext MUST be NULL.")]
    string DesignatedUserContext;
 
+      [Description( "This property is only supported for z/OS! "
+        "When the ShareAS property is set to false, the provider "
+        "module will run in its own copy of a Provider Agent process. No other "
+        "provider module will be loaded into this process. "
+        "Setting the ShareAS property to false has a major impact on the "
+        "performance characteristics, so it should be set to true unless there "
+        "is an urgent need for a provider module to be protected from other "
+        "provider modules running in the same process. For this reason by default "
+        "the ShareAS flag is set to true. "
+        "Setting ShareAS to false is only honored by the CIM Server, when it is "
+        "running with the configuration property for forceProviderProcesses also "
+        "set to true. ")]
+   boolean ShareAS = true;
+
       [Description ( 
         "   Indicates the current status(es) of the element. "
         "Various health and operational statuses are "
         "defined. Many of the enumeration's values are self-"
         "explanatory.  However, a few are not and are described "
         "in more detail. \"Stressed\" indicates that the element "
         "is functioning, but needs attention. Examples of "
